main/main.cpp

```diff
@@ -1020,6 +1020,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	}
 
 	Engine::get_singleton()->set_iterations_per_second(GLOBAL_DEF("physics/common/physics_fps", 60));
+	ProjectSettings::get_singleton()->set_custom_property_info("physics/common/physics_fps", PropertyInfo(Variant::INT, "physics/common/physics_fps", PROPERTY_HINT_RANGE, "1,120,1,or_greater"));
```

Comment:
144 Hz is already a common refresh rate for some monitors, so I guess the range should go up to 144. (With `or_greater` it's already possible to go beyond, but I would expect the default slider range to include all common monitor refresh rates.)

Reply:
Though I see `force_fps` is limited to 120 below too, so I guess it's fine as is :)